diff -r -U 3 minecraft/com/mojang/minecraft/CanvasMinecraftApplet.java minecraft_patched/com/mojang/minecraft/CanvasMinecraftApplet.java
--- minecraft/com/mojang/minecraft/CanvasMinecraftApplet.java	2021-10-17 13:27:56.410804900 -0400
+++ minecraft_patched/com/mojang/minecraft/CanvasMinecraftApplet.java	2021-10-17 13:27:07.686412800 -0400
@@ -8,7 +8,6 @@
 
     CanvasMinecraftApplet(MinecraftApplet var1) {
         this.applet = var1;
-        super();
     }
 
     public final void addNotify() {
